examples/reference-entity-model/input/xmlCustomersWithRelatedEntities/customer1WithRelatedEntities.xml

```diff
@@ -0,0 +1,79 @@
+<Customer xmlns="customerInfo">
+  <CustomerID>401</CustomerID>
+  <Name type="preferred">
+    <FirstName>Jane</FirstName>
+    <LastName>Seymour</LastName>
+  </Name>
+  <Nicknames>
+    <nickname>Janie</nickname>
+    <nickname>Janey</nickname>
+  </Nicknames>
+  <Address>
+    <Shipping>
+      <Street>Covington Road</Street>
+      <City>Los Altos</City>
+      <State>California</State>
+      <Postal>94024-0241</Postal>
+    </Shipping>
+    <Billing>
+      <Street>Latham St</Street>
+      <City>Mountain view</City>
+      <State>California</State>
+      <Postal>94040-6783</Postal>
+    </Billing>
+  </Address>
+  <Orders>
+    <Order>
+      <OrderID>3002</OrderID>
+      <DateAndTime>2019/05/24-07:45:22</DateAndTime>
+      <FirstName>John</FirstName>
```

Comment:
Why does Order have FirstName and LastName?

Reply:
I just based it on the example document in https://wiki.marklogic.com/display/PM/Mapping+Example+%28Data+Hub+5.5%29%3A+Map+multiple+entities+from+a+single+source. It's not used and I can remove them.

Reply:
Yeah we want the example data and the mappings to be as lean and mean as possible - it makes the example as easy as possible to understand removes any noise from confusing folks.